.travis.yml

```diff
@@ -6,18 +6,16 @@ env:
     # GitHub ("Packaging") API token for citus-bot
-    - secure: "uEIo5Gl2T6/4MSTay4Wioo8P2xuuymcLV0H/66JZGFoglwR+2okuphUKikmiayjskksvpChDyEebuju//J1Il0TDZ+lk706cKulQP0rrqwZc9o9bpb0Pq9WOgPORQ2LjHA7fO5/k+jpSQQic67Zyvtbv112Py43NZ/xAEV554WcAkztEpFG8lyajTZZopMjNxyzswBv7CXcvAZDc2LvZJ/caqkVzzozI6Zw8XuP4reHfNW97M/PqeX34JibHa2rRhKJnc9fDiCcXlGleOFbkwlUO1pHQY8aSreemmMJhhx20xtpKlLrTzTXN9VMNzx/3uiFryg1vQvzJRtC5P6RlteUMnjJ3BYPMxUEdeAcz2/O3z1mIBtjbcvZdUKzQYMbMwl/Wn+7uI89N36THGq39SYjjqCSeWdP+VgJ8dPy5MfE4wB4qwkPBC1eljGWe0RtAy7Op9O+13PPeQ+ZhtbB+3IW7aWdmFUFGyOmtOYULzksqCdKjoW/sCBuFymbwH3LUkBQBDfNZHWEGUQCam4ZzYckvCHv0fBzPwkRCpknhSgRKtZuu/C8m3yxns2lhvFR8PFHi9uMoXQOndP4H6qrVHSftga3tRRDJUZryF/A9XrGo3I8VwumVOX9tIiQviuwUyMtaagRzmoHNfUU6T+7sjFjtb7Auw3s84gVnRsqGDYQ="
+    - secure: "zb6AHEUEvpCnWb4zRTsWTWVmMWNHkzz4c+UUgABaXYysJHCfBNiu5X4Da/jusCPY9GH4XFX8GPCiR69ZKw/LDnzHgXoH2pmda02CZM6SHjRGd0MrlBl0hF8YbHiBFi6/nH7kF0ODX+SepfJlFq8LQq6HVONz6QQl0FW0Rg1qnNaDMMi6L9NBBSDyuYCUdY05lgNeIyG8LyIErvlNA8U86VxOHp/KKbNaNx0Cs/tpDWNeW1v1rxa6EXtt8pxyc4u09V9csyFcglizkWEi98aqBTqyCa4hgIK3anog51vAg997cPOUu+ENUpwsio2JMorXalzNszBtpbO9u15+hLRHAoSL56hRdar3fTCuyyXpKapyNZ5Xxa8bYmgLDSJ2lS3zDl6fUfJWVqfzDWx1BuYVGjmJquYMVGKDiBu6iiOx7M/L8Kwof/ioe1LWro92a+S2KKgmXJfxNm4//F2EK+n++GNMZzOsfW+HSVhZ5sXt/tN9eOGmRQ4jWOZK7ycdpB+zMtxYDur/D46oLGAJhY88YqJaBbx0Jk0EMzSfM1eSoGcXEfLfwzGTm3MFnG/IJ3BPW1pkf5s2gMofUssGFitqnDZ7wlexrwA6qPMPIafW+yn5m4wwQwAxk20Y/kOREKHYDP6X6AbQNmfUk93/djYLjd/gGYLS3xvkNOLBWq2CCD8="
     # passphrase for packaging gpg secret key
     - secure: "pZmrxStWj0vXaG/YAyJORvxfaKQHvr+O+66yjFF52gQTVd2zNb83Mn4zDLNqMwU3nkXEKYuFbpXKbDSMX9zNbE7W/w1ARSa8MVXnMVM3jbj+OCaBu7fyVrA0tvT3EZkMh943paTRcSyPSzd61P3CNOkcY1RycDHBbJtNrzSdUbdAy6//TWgNxC66dxSeHASRJIIs49caxQssMMSGRgC5aGH4KLU5eSdP3wnWDEHr+aLqaI2DF2W3lNMI0WJwtGN6cLEYzAsA3BkoPoYRFDd4jjTpolVdrFBXRaT1EIg0FXjsvjyFrQfQLBIqvSx42H5w4rCIYoKYYgRgCKDATLK8XxPbFsgPKLVSHFD6t9ebuYacYxS20oMakkqj5e98qMIWe2vkm+V9YNeADIlhYT+NsF9Kug2B6pLK+U+Hhj6TldFpP18snPXttRKzsg9QrGbnBUVq2wv60U7/3pWs+T3tpr9bIFyaUYbRRAwl/hv2hVkFiSdsiLB6tJphbILB7EDKBCu1iK9PC9tlYoGrqptRxj6bmlawOvcnEKRtoPNJ9eyrvHPDd8FmmW6xWbq89zJ97Ztl7VR8CzNePqvG9vwdtJigXovuXfzu4CHX+jOt3jvllB6rMprsK5r++qBzh1SS24sYqo/APtUfoYdkleWhC+tSuhbucZb2yGUCMo/ZhDo="
   matrix:
     - TARGET_PLATFORM=el/7
-    - TARGET_PLATFORM=el/6
     - TARGET_PLATFORM=ol/7
-    - TARGET_PLATFORM=ol/6
 before_install:
-  - git clone -b v0.7.7 --depth 1 https://github.com/citusdata/tools.git
+  - git clone -b v0.7.13 --depth 1 https://github.com/citusdata/tools.git
```

Comment:
@hanefi when you fix this for the other platforms, can you fix it here, too?

Reply:
ok, taking a mental note. Once I get in the docker hub org, I will update tools version in all `$distro-$project` branches.

I guess this will happen during 9.1 packaging